client/utils/activity/render.ts

```diff
@@ -10,5 +10,5 @@ export const renderActivityItem = (
 	// This cast avoids a TypeScript error:
 	// Expression produces a union type that is too complex to represent. ts(2590)
 	const renderer = activityItemRenderers[item.kind] as ActivityItemRenderer<any>;
-	return renderer(item, context) as RenderedActivityItem;
+	return (renderer?.(item, context) as RenderedActivityItem) || null;
```

Comment:
❗️Looking at my IDE I don't think this is necessary (and I also don't know how TS is accepting `null ∈ RenderedActvitiyItem` with our compiler settings! Could you (1) try reverting this and see if it's okay and (2) if not, add a `!` non-null assertion to the extraction of `renderer` on the previous line?

My high-level goal here is for callers and readers of this function to be confident that they'll unconditionally get a `RenderedActivityItem` for any `ActivityItem`.

Reply:
So Gabe discovered this issue as he was testing this PR. The activity dashboard page was crashing because the `activityItemRenderers` map didn't have an entry for a submissions-related activity item. This was a quick fix without having to block testing/merging this PR until we had renderer(s) written for the culprits. Would you like me to implement those as part of this PR?

Reply:
Oh, I should have seen that. Don't worry about it for now — I'll make a card to fix this up on the project board.

Reply:
Sounds good :D